### PR TITLE
Addition of Source Terms for NEMO Axisymmetric Flows

### PR DESCRIPTION
**Pull request by jtneedels**
## Proposed Changes
 

- Additional source terms are implemented in the residual computation for NEMO planar axisymmetric flows. This allows inviscid and viscous axisymmetric simulations. 

## Related Work

- Built upon framework in PR #1106 by FlorianDM, which introduced viscous axisymmetric source terms for non-reacting flows.

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags, or simply --warnlevel=2 when using meson).
- [x] My contribution is commented and consistent with SU2 style.
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp) , if necessary.

### CODE REVIEW

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (WallyMaier) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:302]: Maybe add a comment here.  Not necessary but I think looks better.   i.e.  "Renaming for conveniences"

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (WallyMaier) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:345]: This spacing looks off.  Extra spaces?

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (WallyMaier) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:362]: you could do something similar here: "su2double qy_ve"

Again, this just aesthetic

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (WallyMaier) [inline SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1199]: maybe add a comment here

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (WallyMaier) [inline SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1232]: Something is off with your spacings here

File: SU2_CFD/src/variables/CNEMOEulerVariable.cpp
Comment (WallyMaier) [inline SU2_CFD/src/variables/CNEMOEulerVariable.cpp:82]: spacings

**Review by WallyMaier — CHANGES_REQUESTED** [review]
@jtneedels thanks for deriving these source terms.  They will be very helpful! 

Please find my comments above

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (jtneedels) [reply SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:302]: Fixed.

Comment (jtneedels) [reply SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:303]: Okay, addressed this.

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (jtneedels) [reply SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1200]: Is this what you mean by a comment here?

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (WallyMaier) [reply SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:303]: Thanks!

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (WallyMaier) [reply SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1200]: Yup!

**Review by WallyMaier — APPROVED** [review]
This looks good to me!  I think you should wait for another approval before pushing.

**Comment by pcarruscag** [comment]
@jtneedels the testcases currently failing are my fault, they will be fixed in a minute.

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (CatarinaGarbacz) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:347]: extra line here

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (CatarinaGarbacz) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:368]: you can just ident this 2 lines for aesthetics (the += part)

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (CatarinaGarbacz) [inline SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1253]: extra line between between the brackets

**Review by CatarinaGarbacz — APPROVED** [review]
Hi @jtneedels . good work. I just made some minor comments, I didn't notice anything "big", so for me it's a GO

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp
Comment (pcarruscag) [reply SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:368]: I would not spend too much time with that kind of alignment, eventually we will run clang-format on the entire code... The only thing you can do to make code look good is to write small functions.

File: SU2_CFD/src/solvers/CNEMOEulerSolver.cpp
Comment (jtneedels) [reply SU2_CFD/src/solvers/CNEMOEulerSolver.cpp:1252]: Thanks for the review Catarina, I think I've addressed this and the other comments on formatting.

**Review by fmpmorgado — APPROVED** [review]

**Pull request merged**
